sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -319,7 +319,9 @@ def __init__(
             self._credential = credential  # type: ignore
         self._auto_reconnect = kwargs.get("auto_reconnect", True)
         self._auth_uri: str
-        self._eventhub_auth_uri = f"sb://{self._address.hostname}{self._address.path}"
+        # remove port (if present) from auth_uri
+        auth_uri_hostname = self._address.hostname.split(":")[0]
```

Comment:
what if `localhost:500` do we need port for auth_uri?

Reply:
is this for proxy ? because you would need the port number there

Reply:
I don't think so, proxy gets passed through http_proxy so it shouldn't get into the fully_qualified_namespace path: https://github.com/Azure/azure-sdk-for-python/blob/ef4ac38750be41f2895579912d4f9df871a064f5/sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py#L312